functional-tests/fn_prover_checkpoint_latest.py

```diff
@@ -0,0 +1,26 @@
+import time
+
+import flexitest
+
+from utils import wait_for_proof_with_time_out
+
+
+@flexitest.register
+class ProverClientTest(flexitest.Test):
+    def __init__(self, ctx: flexitest.InitContext):
+        ctx.set_env("prover")
+
+    def main(self, ctx: flexitest.RunContext):
+        prover_client = ctx.get_service("prover_client")
+        prover_client_rpc = prover_client.create_rpc()
+
+        # Wait for the Prover Manager setup
+        time.sleep(60)
```

Comment:
I am a little concerned if these functional tests are run in CI. The CI will take forever to complete. :D

Reply:
Oh yes.